pkg/sql/schemachanger/scdecomp/decomp.go

```diff
@@ -268,6 +268,11 @@ func (w *walkCtx) walkRelation(tbl catalog.TableDescriptor) {
 		ObjectID:       tbl.GetID(),
 		ParentSchemaID: tbl.GetParentSchemaID(),
 	})
+	if tbl.IsPartitionAllBy() {
+		w.ev(descriptorStatus(tbl), &scpb.TablePartitionAllBy{
+			TableID: tbl.GetID(),
+		})
+	}
```

Comment:
This element is not going to age well at all when we actually are going to have to add PARTITION ALL BY support. Consider replacing this with something named `TablePartitioning` which is like `IndexPartitioning` but without the IndexID?